src/parsers/i3d/main.ts

```diff
@@ -118,11 +135,24 @@ function unpackData(
     sector.instancedMeshGroup.createTreeIndexMap();
   }
 
-  const sectors = maps.idToSectorMap;
   for (let treeIndex = 0; treeIndex < maps.treeIndexNodeIdMap.length; treeIndex++) {
     const nodeId = maps.treeIndexNodeIdMap[treeIndex];
     maps.nodeIdTreeIndexMap.set(nodeId, treeIndex);
   }
+}
+
+function findLargestTreeIndexInMeshes(compressedData: PerSectorCompressedData) {
+  let largestTreeIndex = 0;
+  Object.keys(compressedData).forEach(sectorPath => {
+    const instancedMeshData = compressedData[sectorPath].instancedMesh;
+    const mergedMeshData = compressedData[sectorPath].mergedMesh;
+
+    [instancedMeshData, mergedMeshData].forEach(data => {
+      if (data !== undefined) {
+       largestTreeIndex = Math.max(largestTreeIndex, data.indices.findLargestValue() + 1);
```

Comment:
Why do you do +1 here? Since we do that, `largestTreeIndex` is always smaller than the actual value.

Reply:
We should also be able to use .filter.reduce here? Or that might not be better? @nsunderland-cognite :P

Reply:
```
    const largestTreeIndex = Object.values(compresssedData)
      .reduce(
        (allData, data) => allData.concat(data.instancedMesh, data.mergedMesh),
        []
      )
      .filter(data => data != null)
      .reduce(
        (largest, data) =>
          Math.max(largest, data.indices.findLargestValue() + 1),
        0
      );
```

This is the best I could think of, but meh. All I could really change from what @Verkhovskaya did is use `Object.values` for simplicity and do the `data != null` check instead of `data !== undefined`